### PR TITLE
Add permission tests for foreman_salt

### PR DESCRIPTION
**Pull request by dosas**
### Problem Statement

Missing foreman_salt permissions

### Solution

Add permissions

### Related Issues

similar to https://github.com/SatelliteQE/robottelo/pull/16892

relevant tests: tests/foreman/api/test_permission.py

### CODE REVIEW

**Comment by Satellite-QE** [comment]
Can one of the admins verify this patch?

**Review by ogajduse — APPROVED** [review]
I triggered a completely internal build of PRT, build no. 9420. Test failures I have seen are not related to this patch.

File: robottelo/constants/__init__.py [hunk 1]
```diff
@@ -1162,6 +1186,7 @@
     ],
     'Realm': ['view_realms', 'create_realms', 'edit_realms', 'destroy_realms'],
     'RemoteExecutionFeature': ['view_remote_execution_features', 'edit_remote_execution_features'],
+    'Report': ['create_reports'],
```
Comment (lhellebr) [inline robottelo/constants/__init__.py:1189]: How is this related to foreman-salt?

Comment (dosas) [reply robottelo/constants/__init__.py:1189]: I cannot answer that satisfactorily, should I remove it?

Comment (lhellebr) [reply robottelo/constants/__init__.py:1189]: I guess so. Maybe that permission SHOULD be there (?), but I guess that's for another PR.

Comment (dosas) [reply robottelo/constants/__init__.py]: removed

**Review by pondrejk — APPROVED** [review]
